pyproject.toml

```diff
@@ -14,7 +14,7 @@ packages = [
 python = "^3.12"
 datasets = "*"
 pandas = "*"
-litellm = "*"
+litellm = { git = "https://github.com/xingyaoww/litellm.git", branch = "xw/fix-fn-call" }
```

Comment:
We need to stick with this until litellm merges my PR: https://github.com/BerriAI/litellm/pull/6425 :(